.rubocop_todo.yml

```diff
@@ -20,14 +20,40 @@ Layout/CommentIndentation:
   Exclude:
     - 'config/environments/production.rb'
 
```

Comment:
I think you'll find, for rubocop, that there are some reasonable patterns in other rails apps of files to exclude from rubocop (e.g. the autogenerated ones, etc.)

Reply:
I just created issue #14 to address rubocop settings.